packages/scripts/src/main/findFaqMain.ts

```diff
@@ -30,26 +35,43 @@ async function main() {
   const client = await MongoClient.connect(MONGODB_CONNECTION_URI);
   try {
     const db = client.db(MONGODB_DATABASE_NAME);
-    const faq = await findFaq({
-      db,
-      clusterizeOptions: {
-        epsilon,
-      },
-    });
     const faqCollection =
       db.collection<WithId<FaqEntry & { created: Date; epsilon: number }>>(
         "faq"
       );
     const created = new Date();
-    const insertResult = await faqCollection.insertMany(
-      faq.map((q) => ({
-        ...q,
+
+    // Associate each question with an ID to track questions across runs of
+    // findFaq
+    await findFaq({
+      db,
+      clusterizeOptions: {
         epsilon,
-        created,
-        _id: new ObjectId(),
-      }))
-    );
-    console.log(`Inserted ${insertResult.insertedCount} FAQ entries.`);
+      },
+    })
+      .then(async (faq) => {
```

Comment:
Any reason to use a promise chain here instead of `await`?

Reply:
Yes (you know I would never use a promise chain without a reason).

In this case I wanted each scope to be self-contained rather than assigning some variable over and over or having a bunch of `await Promise.all()`s. So I'm using the Promise.then() as a pipeline.